tests/project-type/workspace-instance.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { WorkspaceImpl } from "@sap/project-api";
+import { expect } from "chai";
+import { uniq } from "lodash";
+
+import { mockVscode } from "../mockUtil";
+
+const testVscode = {
+    workspace: {
+        workspaceFolders: () => []
+    }
+};
+
+mockVscode(testVscode, "src/project-type/workspace-instance.ts");
+import { getBasWorkspaceAPI } from "../../src/project-type/workspace-instance";
+
+describe.skip("getBasWorkspaceAPI", () => {
+    it("multiple bas workspace instances should be equal", () => {
+        const basWsInstances = [];
+        for (let i = 0; i < 5; i++) {
+            basWsInstances.push(getBasWorkspaceAPI(WorkspaceImpl));
```

Comment:
This is completely **subjective**, but I would do a one liner to initialize this `basWsInstances` array.
```javascript
const {range, map} = require("lodash")
const basWsInstances = map(range(0,5), () => getBasWorkspaceAPI(WorkspaceImpl))
```

If we remove the constructor parameter from `getBasWorkspaceAPI` it becomes even shorter.

```javascript
const {range, map} = require("lodash")
const basWsInstances = map(range(0,5), getBasWorkspaceAPI)
```

The reason it is **subjective** is that is general style differences between imperative and functional programmatic styles.
On the other hand we do use lodash semi functional style a-lot so it may fit our project.

Reply:
Done